src/main/java/seedu/address/model/task/Location.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.commons.Name;
+
+public class Location {
+    public final String location;
+
+    public Location(String location) {
+        requireNonNull(location);
+        this.location = location;
+    }
+
```

Comment:
Should we use Address Class for Task as well? I am unclear if there is a difference between Location and Address class class haha

Reply:
Hmm I have removed this class already?

Reply:
Ah okay not yet, my bad